src/main/java/pl/sda/eventmanager/model/Event.java

```diff
@@ -0,0 +1,105 @@
+package pl.sda.eventmanager.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+import java.util.Set;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Event {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long Id;
+
+    @ManyToOne(fetch = FetchType.EAGER)
+    @JoinColumn(name = "USER_ID")
+    private User eventOrganiser;
+
+    @Column(nullable = false, unique = true)
+    private String eventName;
+    @Column(nullable = false)
+    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
+    private LocalDateTime eventStart;
+    @Column(nullable = false)
+    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE_TIME)
+    private LocalDateTime eventEnd;
+    @Column(nullable = false)
+    private String eventDescription;
+
+    @ManyToMany(mappedBy = "attendedEvents")
+    private Set<User> eventAttendants;
+
+
+
+    public static final class EventBuilder {
```

Comment:
no need to create that manually, you can use `@Builder` on `Event`, if you decide to keep that class for whatever reason, please remove unused methods `anEvent()`